lib/matplotlib/offsetbox.py

```diff
@@ -136,21 +132,30 @@ def _get_packed_offsets(wd_list, total, sep, mode="fixed"):
 
 def _get_aligned_offsets(hd_list, height, align="baseline"):
     """
-    Given a list of (height, descent) of each boxes, align the boxes
-    with *align* and calculate the y-offsets of each boxes.
-    total width and the offset positions of each items according to
-    *mode*. xdescent is analogous to the usual descent, but along the
-    x-direction. xdescent values are currently ignored.
+    Vertically align boxes each specified by their ``(height, descent)`` pair.
```

Comment:
```suggestion
    Align boxes each specified by their ``(height, descent)`` pair.
```
Per above (not as strict because it still has direction mentioned in the tuple, but "Align boxes" would maybe a bit short).

Reply:
yes